### DISCUSSION TITLE
zh/guide/

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#19] [Announcements]
# zh/guide/

AList v3的文档

https://alist.nn.ci/zh/guide/

**Comment by 3555555678** [comment]
顶楼 支持 支持 支持 支持 支持

**Reply by humin899** [reply]
突发奇想。如果主流的网盘，支持分享链接转存。就不用登录自己的账号这么麻烦了

**Reply by dr34m-cn** [reply]
https://github.com/dr34-m/taosync

支持alist挂载的各种存储之间自动定时同步，欢迎各位试用、点star，新项目急需star！

**Comment by Animism001** [comment]
windows v3版用vbs启动提示密码错误……
Set ws = CreateObject("Wscript.Shell")
ws.run "alist.exe server",vbhide
用别的隐藏终端命令窗口的方式也是不行
if "%1"=="hide" goto CmdBegin
start mshta vbscript:createobject("wscript.shell").run("""%~0"" hide",0)(window.close)&&exit
:CmdBegin
alist server

alist程序已经添加到系统环境变量里了
我用上面的方法运行alist，用原来的用户名和密码登录提示用户名和密码错误，用admin用户名登录则是密码错误，然后用命令行终端运行alist admin显示的又是原来的用户名与密码……晕了

**Reply by at86397388** [reply]
设置是保存在data.db文件里的，所以只要第一次设置好，然后备份data文件，以后被初始化了，只需要把data文件替换，再重启服务就可以了

**Comment by Animism001** [comment]
应该是在alist应用程序目录下的data和log两个文件对吧？
可是win11的话（其他系统没试过）用ps运行alist server命令，它会在users/用户名路径下创建data和log这两个文件夹的，数据也是保存在这里面。（我要的）
而用vbs或bat后台运行的时候，读取的数据有变成alist路径下而data和log了（软件再次创建的，新的，不是我要的）
就跟原来设置好的users/用户名路径下没关系了，但是运行alist admin读取的又是users/用户名路径的数据（前面我要的），就读取不到vbs启动创建的数据了……
我现在的解决方法就是把alist程序放在users/用户名路径下，这样不管那种方式启动，读取的都是同一份数据了，就是users/用户名路径的内容会显得比较杂乱难看……另一种方法就是用ps运行alist server，在网页上设置好一切之后，之后，将users/用户名路径下的data和log复制到alist同一文件夹中。这样的问题就是每次改动什么就要复制来复制去的……

还有一个问题就是win11运行alist一段时间后，alist.EXE会被删掉，自带的杀毒软件也没提示是它删的，defender记录里也没有删除的记录，我都不知道到底是谁把它删了！见鬼了！

从 Windows 版邮件<https://go.microsoft.com/fwlink/?LinkId=550986>发送

发件人: ***@***.***>
发送时间: 2022年10月14日 22:05
收件人: ***@***.***>
抄送: ***@***.***>; ***@***.***>
主题: Re: [alist-org/docs] zh/guide/ (Discussion #19)


设置是保存在data.db文件里的，所以只要第一次设置好，然后备份data文件，以后被初始化了，只需要把data文件替换，再重启服务就可以了

―

**Reply by at86397388** [reply]
我现在遇到的问题是

为什么我在阿里云盘转存了视频，通过alist里却看不到，一定要在当前目录点右边的刷新才会出来？目录没有自动刷新，

**Reply by 3805139QQ** [reply]
需要时间 同步刷新才行，不是自己的硬盘没那么快

**Reply by leonwahy** [reply]
一样发生了莫名其妙不见得问题。更新后问题太多，费解。

**Comment by at86397388** [comment]
请问
为什么我在阿里云盘转存了视频，通过alist里却看不到，一定要在当前目录点右边的刷新才会出来？目录没有自动刷新，

**Reply by hnnyxycsy** [reply]
它本来就不会自动刷新的，必须手动

**Reply by Yzz-Ymin** [reply]
不是的，配置里面说了嘛，那个缓存时间就是目录的有效时间，你设置为0应该就行了吧，或者小一点

**Reply by mlkt** [reply]
页面当然不会自动刷新啊，谁知道你的文件什么时候会变化，为了极少数情况难道要每个几秒自动刷新

**Comment by at86397388** [comment]
Alisa ipa程序哪里可以下载呢

**Reply by Animism001** [reply]
没有ipa，他说的应该是从alist搭建的网盘里面下载ipa文件，可以直接在iPhone上安装

**Comment by wer727** [comment]
这个要怎么下载求大佬指点

**Reply by Animism001** [reply]
alist文档 安装 里面有教程和下载地址啊

**Comment by at86397388** [comment]
能支持 亲宝宝 么？
https://www.qinbaobao.com/?bd_vid=10140611516743844326

**Comment by xj9696** [comment]
能加支持TeraBox吗？

**Comment by KimiMeta** [comment]
您好，请问有支持ARM7梅林380固件的版本吗？

**Comment by kuaisubushu** [comment]
天翼云盘当上传大文件时提示错误，CloudDrive挂载通过扫码登陆，可以上传大文件，连接客户端驱动有限制

**Comment by Wang320281** [comment]
请问能不能添加支持115网盘。。。。。

**Comment by kuaisubushu** [comment]
由于天翼云盘和移动云盘 也采用referer 的限制，建议像阿里云盘那样增加使用移动端 token，扫描二维码获取方式添加

**Reply by kuaisubushu** [reply]
https://miniapp.cloud.189.cn/doc/develop/server/mp/ecloud/getECloudToken.html调用天翼云盘api能力

**Comment by flyinhere** [comment]
请在在威联通DOCKER中如何更新程序版本？

**Comment by yws112358** [comment]
一个小问题：管理后台，左下角有个切换语言功能，日语貌似就是英语

**Comment by simon333888** [comment]
115网盘，隔段时间就提示要重新登录，有没有好的解决办法呢？

**Reply by zi4dbek** [reply]
没办法

**Reply by dilemma-ske** [reply]
怎么设置开机自启啊，求大佬指导

**Comment by alrightw** [comment]
挂载后输入端口号不用输入用户名和密码就能看到挂载内容，如何设置隐私？

**Reply by RiverGcG** [reply]
同问，直接输入ip+端口后就能以admin的身份进入了，完全绕开了登录

**Comment by xuanyu1937** [comment]
请问edge浏览器访问无论是官方演示站点还是我自己的都显示System error: SyntaxError: Invalid regular expression: /?/: Nothing to repeat怎么解决

**Comment by yechaogui** [comment]
支持支持

**Comment by ywlpostman** [comment]
如何把自动播放音频时默认的随机播放改为按顺序播放？我的意思是不需要每次播放音频时都要点一下，家里老人听下载好的评书，每次随机手又哆嗦点不到调整播放的按键。拜托了，望解答一下。

**Reply by ywlpostman** [reply]
谢谢您的回复，docker有什么办法吗？

**Comment by chen360781** [comment]
蓝奏云挂载后存文件被拒绝怎么办

**Comment by xinxinenjoy** [comment]
请教，部署好之后怎么样才能让内网其他设备直接访问？
还是说它就无法实现，只能部署了的这台机器才能访问到

**Reply by Animism001** [reply]
部署之后内网其他机器都能访问，访问地址为部署了alist的机器ip加端口（192.16.x.x:5244）

**Reply by xinxinenjoy** [reply]
其他机器是需要通过SMB还是WEBDAV方式进行访问？  我试验了以后发现都不行，方便的话能否加您个联系方式详细请教一下

**Reply by GHOST1834** [reply]
请问使用内网穿透搭建的alist要怎么挂载呢？

**Comment by Animism001** [comment]
直接浏览器访问就行ip:端口，smb不行，webdav可以，webdav地址是ip:端口/dav，用户名和密码是你设置的



发送自我的盖乐世



-------- 原始信息 --------
发件人： xinxinenjoy ***@***.***>
日期: 2022/12/21 下午4:30 (GMT+08:00)
收件人： alist-org/docs ***@***.***>
抄送： Animism001 ***@***.***>, Comment ***@***.***>
主题： Re: [alist-org/docs] zh/guide/ (Discussion #19)


其他机器是需要通过SMB还是WEBDAV方式进行访问？ 我试验了以后发现都不行，方便的话能否加您个联系方式详细请教一下

―

**Comment by xinxinenjoy** [comment]
这个账号密码难道不是alist的，而是需要单独再设置一个？我用的是alist的账号密码webdav链接不上来自手机的邮件在 2022年12月21日，17:54，Animism001 ***@***.***> 写道：﻿
直接浏览器访问就行ip:端口，smb不行，webdav可以，webdav地址是ip:端口/dav，用户名和密码是你设置的







发送自我的盖乐世







-------- 原始信息 --------

发件人： xinxinenjoy ***@***.***>

日期: 2022/12/21 下午4:30 (GMT+08:00)

收件人： alist-org/docs ***@***.***>

抄送： Animism001 ***@***.***>, Comment ***@***.***>

主题： Re: [alist-org/docs] zh/guide/ (Discussion #19)





其他机器是需要通过SMB还是WEBDAV方式进行访问？ 我试验了以后发现都不行，方便的话能否加您个联系方式详细请教一下



―

**Comment by Animism001** [comment]
你怎么部署的，部署之后在本机上用浏览器能访问吗？你没自己设置密码那用默认的用户与密码就能链接。webdav你用的是什么软件？看看是不是域名甚至错误。要不然就是你的设备根本不在同一个网段，可能你用了多个路由器，又没设置好，多试几次看看

**Comment by xinxinenjoy** [comment]
本机上可以通过IP加端口号直接进到页面里，访问网盘，webdav想连接的是电视机上的Kodi，网段无问题正常可以访问电脑上共享的其它内容这个账号是登录alist的那个账号？还是进去管理界面之后新增来自手机的邮件在 2022年12月21日，18:03，Animism001 ***@***.***> 写道：﻿
你怎么部署的，部署之后在本机上用浏览器能访问吗？你没自己设置密码那用默认的用户与密码就能链接。webdav你用的是什么软件？看看是不是域名甚至错误。要不然就是你的设备根本不在同一个网段，可能你用了多个路由器，又没设置好，多试几次看看

**Comment by Animism001** [comment]
用户密码这个没问题，应该是你kodi设置的问题，我没用过kodi链接webdav，不过我想问题可能是你在kodi上输入地址时没有在后面加上/dav，类似192.168.x.x:5244/dav，是你kodi设置的问题，你部署的alist没问题。

**Comment by xinxinenjoy** [comment]
就很神奇，kodi这边可以确定设置的没问题会不会有其他东西的限制，比如路由模块的某项设置？来自手机的邮件在 2022年12月21日，18:58，Animism001 ***@***.***> 写道：﻿
用户密码这个没问题，应该是你kodi设置的问题，我没用过kodi链接webdav，不过我想问题可能是你在kodi上输入地址时没有在后面加上/dav，类似192.168.x.x:5244/dav，是你kodi设置的问题，你部署的alist没问题。

**Comment by Animism001** [comment]
你有安卓机就下载个文件管理器之类的能用webdav的，然后试一试你部署的alist的webdav服务，能用就说明没问题。

可能就是你kodi的设置问题。我看了网上kodi的webdav设置界面，不知道你的是英文还是中文，正确的设置应该是服务地址填上http://ip，远程路径填dav（也可能是、/dav）,端口5244，用户与密码填你alist的，还不行我就不知道了。不同软件连接webdav的界面不一样，这个需要你自己尝试了，ip,端口,远程路径这三者ABC是可以拆分组合的。192.168.*.*（A）:5244（B）/dav（C），你多试几次

**Reply by xinxinenjoy** [reply]
测试过了，只有本机可以访问，局域网内其他设备都不行  又是毫无进展的一天

**Comment by xinxinenjoy** [comment]
有道理，一会儿用手机试试

来自手机的邮件

**Comment by Animism001** [comment]
用你的手机浏览器输入alist的访问地址试试，或者看看是不是前面没加http://，webdav不是smb，不是直接输入ip就行



发送自我的盖乐世

**Reply by xinxinenjoy** [reply]
专门给电脑也装了KODI，同样的配置可以直接访问到网盘内容。目前看来是局域网设备无法通过部署的PC来访问Alist的内容

**Reply by xinxinenjoy** [reply]
只有电脑可以，电视，手机（安卓IOS）都不行

**Comment by Animism001** [comment]
我就是电脑部署，手机访问，没有任何问题，不行你就看看ip是不是输入正确，你的电脑共享之后都能通过smb访问文件了，webdav应该也没问题才对。要不然就是你的设备根本不在一个网段，检查一下自己的网络设置吧



发送自我的盖乐世

**Comment by Animism001** [comment]
把你的电脑kodi配置界面截图我看看



发送自我的盖乐世

**Reply by xinxinenjoy** [reply]
要不您加我个Q？这样沟通有点麻烦哈哈
一六八020四二一

**Comment by xinxinenjoy** [comment]
捣鼓两天可算解决了，问题出在防火墙的出入站规则上。添加alist并且把入站规则全打开，竟然就这样通了

**Comment by KJFKA** [comment]
各位大佬infuse通过webdav连接alist挂载的阿里云盘，播放视频卡顿或者一直转圈，很长时间才能播放视频怎么解决？

**Reply by Animism001** [reply]
看看阿里云盘用的是什么代理方式，用302重定向试试，infuse直接通过文件功能进入到阿里云盘webdav选择视频播放，再不然就是你的网络问题，又或者视频太大。这跟alist没关系

**Comment by Animism001** [comment]
运行alist admin读取的是users/用户名/data/data.db的数据，将data文件夹整个复制到alist.exe所在的路径下应该就能解决你的问题

**Comment by axixa120** [comment]
怎么更改alist默认视频播放器啊

**Comment by Marsdancer** [comment]
能支持坚果云吗？

**Comment by JayHawSG** [comment]
支持！！！

**Comment by humin899** [comment]
如果主流的网盘，支持分享链接转存。就不用登录自己的账号这么麻烦了

**Comment by inbytebeat** [comment]
想问一下 为什么阿里云盘的大小文件都可以被IDM接管正常下载，为啥百度云盘的无论大小文件用idm下载完成后是几十kb的奇怪的文件

**Comment by fengchen-github** [comment]
文档中貌似没有写在哪里进入配置页，本来以为在config文件中
看了别人的教程才发现从http://ip:5244/点管理，或者直接http://ip:5244/@manage
希望可以加上这句话，对新手更友好 ，哈哈

**Comment by neverrainmyself** [comment]
为什么没有奶牛快传 cowtransfer.com

**Comment by ffdnm** [comment]
为什么打不开啊显示这个
failed get storage: can't find storage with rawPath: /

**Reply by lisongone** [reply]
我升级到3.10.1后也出现这个问题，应该是现出BUG了，望解决

**Comment by seamountzh** [comment]
怎么才能将之前的alist账户删除，因为我在存储里弄的时候，目录一不小心加了  \123,这个符号后，在管理里无法刷新出挂在的网盘，一直提示“”“No such column: index”，，在主页的时候，\123    这个目录也无法删除，无法访问，一直报错

**Comment by 9390041** [comment]
挂载三播放器不好用了

**Comment by yuanchongfu** [comment]
阿里云盘视频昨天打不开了，显示连接失败；win10系统，是什么原因

**Comment by lisongone** [comment]
您好，我已经收到您的邮件，暂时无法亲自回复。

**Comment by dilemma-ske** [comment]
怎么设置开机自启啊，求大佬指导

**Reply by cxapk** [reply]
安装后自动设置系统变量

**Comment by RmondJone** [comment]
升级了，最新版，手机浏览器播放不了了。然后挂WebDAV也没有之前流畅了，总体感觉还不如旧版。但是不升级阿里云盘又挂载不了，蛋疼的一逼，望作者早日修复，要不然真的没法用。

**Comment by alinjx** [comment]
移动和电信都有了，唯独缺了联通

**Comment by pda8888** [comment]
大佬，能否支持天翼企业云盘？
天翼个人云盘只有区区30个G，天翼企业云盘有200个G，还可以增加，可惜目前（V3.13.2）好像不支持天翼企业云盘。

**Comment by qwwzi** [comment]
用户那边的访客设置，没有勾选 无需密码访问，还能随便打开

**Comment by ailixluo** [comment]
请问一下
登录alist后报
System error: SyntaxError: Invalid regular expression: /?/: Nothing to repeat  怎么搞

**Reply by xiqiliuse** [reply]
建议把油猴插件关掉试试

**Comment by VanYe666** [comment]
迅雷网盘挂载好之后重启就会掉线需要重新登录是怎么回事

**Comment by azureplanet** [comment]
~~非常好alist 爱来自蔚蓝行星~~

**Comment by ponponon** [comment]
哈哈

**Comment by 1458313309** [comment]
支持

**Comment by rufo2015** [comment]
用IDM下载时提示出错:
直接点击提示:
{"error_code":31362,"error_msg":"sign error","error_info":"","request_id":676279172298669748}
把链接复制到IMD下载,提示:
服务器响应显示您没有权限下载此文件,备注:HTTP/1.1 403 Forbidden

如果用来下载几十M的文件是正常的.不知哪里出了问题.

**Comment by rufo2015** [comment]
用IDM下载时提示出错:
直接点击提示:
{"error_code":31362,"error_msg":"sign error","error_info":"","request_id":676279172298669748}
把链接复制到IMD下载,提示:
服务器响应显示您没有权限下载此文件,备注:HTTP/1.1 403 Forbidden

如果用来下载几十M的文件是正常的.不知哪里出了问题.

**Comment by zaoshuishentihaoa** [comment]
1111

**Comment by cyw0012** [comment]
666

**Comment by newens** [comment]
大神，能否增加支持联通云盘？现在的联通云盘真心不错，价格不高，空间够大，云硬盘功能很强。
https://pan.wo.cn/

**Comment by lisongone** [comment]
您好，我已经收到您的邮件，暂时无法亲自回复。

**Comment by GHOST1834** [comment]
请问用安卓手机+内网穿透搭建的公网AList要怎么挂载在其他设备上？

**Comment by daewdfg** [comment]
挂载123云盘 状态一直显示  状态:  当前登录操作频繁，请稍后再试  这要怎么操作 ，求大佬指点，挂是卦上去了 但是就不是Work 状态，

**Comment by xiqiliuse** [comment]
点击文件下载后会跳至这个链接http://img.xiaoya.pro/abnormal.png，无法下载相应文件

**Comment by dddundertale** [comment]
good

**Comment by HumblePolar** [comment]
可以使用ipv6吗？

**Reply by geol** [reply]
我家里没有公网ip，用的ipv6，能访问。

**Comment by geol** [comment]
在手机端浏览器添加储存，下拉的储存列表全挤在一起了，选中目标网盘极其困难。

**Reply by geol** [reply]
补充一哈，ios的Safari浏览器及Alook浏览器都是如此。临时试了下安卓的火狐浏览器，没有出现这个问题。

**Comment by whao123yahoo** [comment]
大佬能否推出安卓模块版本的alist😂

**Comment by GHOST1834** [comment]
安卓可以用termux跑alist，用xlist做客户端或者有些播放器也可以挂alist



---Original---
From: ***@***.***&gt;
Date: Sun, Jul 30, 2023 23:12 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [alist-org/docs] zh/guide/ (Discussion #19)




 
大佬能否推出安卓模块版本的alist😂

**Comment by whao123yahoo** [comment]
termux需要一直挂后台😭

发自我的小米
在 GHOST1834 ***@***.***>，2023年7月30日 23:14写道：

安卓可以用termux跑alist，用xlist做客户端或者有些播放器也可以挂alist



---Original---
From: ***@***.***&gt;
Date: Sun, Jul 30, 2023 23:12 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [alist-org/docs] zh/guide/ (Discussion #19)





大佬能否推出安卓模块版本的alist😂

**Comment by cn-oalo** [comment]
好家伙 ，刚更新就发布新版本 太巧了

**Comment by cxapk** [comment]
更新的快，很正常。。
发自手机版Outlook（https://cxapk.com）



---Original---
From: ***@***.***&gt;
Date: Mon, Aug 7, 2023 17:38 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [alist-org/docs] zh/guide/ (Discussion #19)




 
好家伙 ，刚更新就发布新版本 太巧了

**Comment by a97115038** [comment]
所有的盘的体验，我都觉得非常卡，可能是对文件夹内文件数量的影响，不知道什么时候支持TG，

**Comment by ghost** [comment]
怎么下载呀

**Comment by feiduikang** [comment]
alist挂载的网盘中的rar文件解压之后在C盘哪里能找到这个文件呢？解压一半C盘空间不够我就点取消解压了，现在在C盘找不到这个文件，但是能看到已经占用了C盘的空间

**Comment by Animism001** [comment]
用什么下载的？打开浏览器看下载的文件在哪，估计就在那个目录中

发自我的 iPhone

在 2023年9月11日，09:20，feiduikang ***@***.***> 写道：

﻿

alist挂载的网盘中的rar文件解压之后在C盘哪里能找到这个文件呢？解压一半C盘空间不够我就点取消解压了，现在在C盘找不到这个文件，但是能看到已经占用了C盘的空间

**Comment by 066886** [comment]
pikpak挂载到alist失效了吗，之前挂载的pikpak网盘都打不开了，pikpak分享也打不开。大佬求助

**Comment by jingshuchun** [comment]
非常感谢

**Comment by VincentYD** [comment]
搭建小雅docker时出现错误 ，求大佬指点
docker.errors.APIError: 500 Server Error: Internal Server Error ("Cannot restart container 42ed2b8132b05b07a2b1f4cd74668dbbfe22a721d2fe8baa39f58d7efd9051c6: failed to create task for container: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: error during container init: error setting cgroup config for procHooks process: bpf_prog_query(BPF_CGROUP_DEVICE) failed: invalid argument: unknown")

**Comment by dahetaoa** [comment]
提问
我的配置环境是Ubuntu下用自动脚本安装了alist，Ubuntu内挂载了一个外接硬盘，我使用alist的本地存储挂载了我的硬盘，但是在上传文件到硬盘的时候，明明上传完成了，但过了很久，cpu的io等待还是非常高，并且集中在一个核心上，具体现象是我的Ubuntu有4个核，其中一个核的io等待高达95%，而这个核心的系统占用却很少，别的核心的占用和io等待也很少，查看磁盘写入，发现几乎没有磁盘正在写入中，有也是很小的速度，所以我怀疑是alist的某个bug导致硬盘实际上没有在读/写，但是还是会占用cpu导致高额的io等待，希望能得到解决😢

**Comment by at86397388** [comment]
感觉crypt不能用了？上传图片可以，但大一点的视频，明明提示成功，但网盘里实际没有东西

**Comment by at86397388** [comment]
感觉crypt不能用了？上传图片可以，但大一点的视频，明明提示成功，但网盘里实际没有东西

**Comment by ZRP1107** [comment]
failed link: failed get link: get download url failed: {"method":null,"url":null,"expiration":null,"size":null,"streamsUrl":null,"content_hash":null,"content_hash_name":null,"file_id":null}
这是什么情况？？？

**Comment by bingkuomiao** [comment]
功能很强大，请问大家支持在断网情况下，访问局域网内的本地盘数据吗？我断开互联网试了一下访问本地的SMB共享是不行的，一定要联网才行。

**Comment by baymax222** [comment]
Failed update storage in database: disk I/O error
这个是啥意思？阿里云open保存的时候出这个

**Reply by baymax222** [reply]
忽略吧应该是我的问题

**Comment by lisongone** [comment]
您好，我已经收到您的邮件，暂时无法亲自回复。

**Comment by leix45** [comment]
支持

**Comment by yangleijapan** [comment]
事实上，我觉得最好的网盘其实是Telegram，无限空间，免费支持最大2g单文件，视频图片可以直接预览。不知道alist什么时候能添加Telegram

**Comment by Animism001** [comment]
telegram网盘怎么用？？？


发送自我的盖乐世



-------- 原始信息 --------
发件人： yangleijapan ***@***.***>
日期: 2023/12/22 上午12:36 (GMT+08:00)
收件人： alist-org/docs ***@***.***>
抄送： Animism001 ***@***.***>, Comment ***@***.***>
主题： Re: [alist-org/docs] zh/guide/ (Discussion #19)


事实上，我觉得最好的网盘其实是Telegram，无限空间，免费支持最大2g单文件，视频图片可以直接预览。不知道alist什么时候能添加Telegram

―

**Comment by magichkk** [comment]
批量重命名的正则有示例吗

**Comment by yf2468** [comment]
服务器是windows server 2008 r2，无法启动。

**Reply by yf2468** [reply]
有兄弟可以解答一下吗？谢谢。

**Comment by keleyun6** [comment]
一键安装指令用不了了吗

**Comment by wte4410** [comment]
有没有大神帮忙,alist挂载的迅雷网盘,看电影只有1.5m的速度根本看不了4k视频怎么回事啊?

**Comment by bswc000** [comment]
为什么刷新按钮点不了了？

**Comment by bbqdea** [comment]
好像，我所有的Alist都打不开了～

**Comment by 209180735** [comment]
1

**Comment by at86397388** [comment]
为什么设置里的公告，我清除了，重启，或者不重启过一会，公告又有了？又成了默认状态

**Reply by at86397388** [reply]
版本为v3.32.0

latest

**Comment by 7788user** [comment]
智齿

**Comment by kufx** [comment]
大佬，想请教一下：

就是alist每触摸一个文件的按钮，都会出现一块蓝色的东西，听其他大佬说，它ui用的图片样式，改成背景样式就解决了，
听说可以用css啥的让它消失，
不知道大佬有空能否研究一下，
怎么能加个啥自定义代码可以解决这个问题吗？
(我感觉如果没有那个蓝色的会更美观一些)
麻烦大佬了

**Comment by hrdiyishenxu** [comment]
1

**Comment by zjy807** [comment]
服务器最低配置要求是什么

**Comment by lisongone** [comment]
您好，我已经收到您的邮件，暂时无法亲自回复。

**Comment by Minearm-RPM** [comment]
:)

**Comment by zhiyang5733** [comment]
https://www.icloud.com/iclouddrive/0bbn7lbSD8c16oMGtNo-_r1tQ

苹果icloud 网盘分享  能不能加个驱动 大佬

**Comment by lisongone** [comment]
您好，我已经收到您的邮件，暂时无法亲自回复。

**Comment by 727089742yyk** [comment]
nihao

**Comment by dunjiaow** [comment]
大佬，首页的这个阿里云盘资源搜索引擎怎么使用，点进去是index.html

**Comment by CarvedWind** [comment]
为什么几年前的老电脑win7 64位系统用不了 它执行完.\alist.exe server之后显示的是乱码 比如0x00之类的 怎么解决啊

**Comment by ranzhigg** [comment]
您好开发者  想问下最新版本mac系统 15下载的官网版本alist 安装后打开提示文件损坏 .是因为系统没有开启允许所有来源安装嘛 尝试了一下新系统好像不能开启这个选项了 是否需要关闭sip安装呀

**Comment by Hyun8888** [comment]
迅雷云盘 提示错误 ailed init storage: ErrorCode: 10 ,Error: aborted ,ErrorDescription: Your operation is too frequent, please try again later 怎么解决呢？

**Comment by ghost** [comment]
[Function]希望添加对360安全云盘的支持
